### PR TITLE
T dependent MMS

### PR DESCRIPTION
**Pull request by rekomodo**
Modification of the simple MMS case to accept an x-dependent temperature. A natural extension of this MMS case would be to do it for the trapping MMS case as well.

### CODE REVIEW

**Comment by RemDelaporteMathurin** [comment]
Let's wait until #31 is merged and we figure out what's causing the compilation failure

File: report/verification/mms/simple_temperature.md [hunk 1]
```diff
@@ -0,0 +1,282 @@
...
+  language: python
+  name: python3
+---
+
+# Simple diffusion with varying temperature
+
+```{tags} 2D, MMS
+```
+
+This is an extension of the simple MMS example with a non-homogenous temperature gradient.
```
Comment (RemDelaporteMathurin) [inline report/verification/mms/simple_temperature.md:20]: Could you add a clickable link to the case?

Comment (rekomodo) [reply report/verification/mms/simple_temperature.md]: got it :+1:

**Review by RemDelaporteMathurin — APPROVED** [review]
I would potentially change the location of this case to be right after the "simple diffusion" case but we'll do it in another PR :-)

**Pull request merged**